README.md

```diff
@@ -63,10 +65,10 @@ var Client = require('3scale').Client;
 
 client = new Client("your provider key");
 
-client.authorize({ "app_id": "your application id", "app_key": "your application key" }, function(response){
+client.authorize({ "service_id": "your service id", "app_id": "your application id", "app_key": "your application key" }, function(response){
   if (response.is_success()) {
     var trans = [{ "app_id": "your application id", "usage": { "hits": 3 } }];
-    client.report(trans, function (response) {
+    client.report(service_id, trans, function (response) {
```

Comment:
`service_id` should be `"your service id"` to be consistent with the `client.authorize` call. There are several examples like this.